src/components/v1/V1Project/modals/V1ConfirmPayOwnerModal.tsx

```diff
@@ -195,43 +229,87 @@ export default function V1ConfirmPayOwnerModal({
             name="memo"
             label={t`Memo (optional)`}
             className={'antd-no-number-handler'}
-            extra={t`Add an on-chain memo to this payment.`}
+            style={{ marginBottom: 0 }}
           >
-            <MemoFormInput />
+            <Form.Item
```

Comment:
**Issue:** we want to avoid nested `<Form.Item`s like this. Can we make it work with a single `memo` item?

Reply:
I jumped in and fixed this :)

Reply:
Oh, i thought that comment was related to the one below, sorry!